queries.sql

```diff
@@ -8,3 +8,95 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+ALTER TABLE animals ADD COLUMN species VARCHAR;
+-- Starting the transaction
```

Comment:
- [ ] Dear @Ellyboi, Good job using the `ALTER TABLE` statement to add the column as requested. But while doing so I notice you have added this into your `queries.sql` file. Please remember everything that creates or changes the table structure should be saved in the schema.sql file. Please consider moving this statement to the `schema.sql` file.